src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/ReferenceFinders.cs

```diff
@@ -21,7 +21,7 @@ internal static class ReferenceFinders
         public static readonly IReferenceFinder Operator = new OperatorSymbolReferenceFinder();
         public static readonly IReferenceFinder OrdinaryMethod = new OrdinaryMethodReferenceFinder();
         public static readonly IReferenceFinder Parameter = new ParameterSymbolReferenceFinder();
-        public static readonly IReferenceFinder Property = new PropertySymbolReferenceFinder();
+        public static readonly IReferenceFinder Property = PropertySymbolReferenceFinder.Instance;
```

Comment:
❓ Why make this different from the pattern? It seems preferable to do the following:

1. Revert this line
2. Remove `PropertySymbolReferenceFinder.Instance`
3. Replace all references to `PropertySymbolReferenceFinder.Instance` with references to `ReferenceFinders.Property`

Reply:
will do.

Reply:
Done.